src/components/Navigation/index.js

```diff
@@ -4,25 +4,30 @@ import Nav from "react-bootstrap/Nav";
 import { NavLink } from "react-router-dom";
 import { useSelector } from "react-redux";
 import { selectToken } from "../../store/user/selectors";
+import { selectUser } from "../../store/user/selectors";
 import NavbarItem from "./NavbarItem";
 import LoggedIn from "./LoggedIn";
 import LoggedOut from "./LoggedOut";
 
 export default function Navigation() {
   const token = useSelector(selectToken);
+  const {role} = useSelector(selectUser);
```

Comment:
spaces `{ role }` your VSCode should be getting this for you, do you have prettier?